### PR TITLE
Change whisper-subtitles-generation.ipynb to genai pipeline

### PR DESCRIPTION
**Pull request by sbalandi**

### CODE REVIEW

**Comment by sbalandi** [comment]
openvino.genai was built on https://github.com/openvinotoolkit/openvino.genai/pull/910

File: notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "\n",
```
Comment (as-suvorov) [inline notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb:13]: <p>Should this output be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (as-suvorov) [inline notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb:13]: <pre class="ql-syntax" spellcheck="false">Line #31.            if segment.end_ts is None:
</pre><p>I think it should be:</p><p><code>if segment.end_ts == -1</code></p><p>according to <a href="https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-57b99dea7bc9b9a0b5b94b77a150999f0be5aefd4f9ce0bfd8c758fbaf8b2de9R27" target="_blank">https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-57b99dea7bc9b9a0b5b94b77a150999f0be5aefd4f9ce0bfd8c758fbaf8b2de9R27</a></p><p>You can reproduce it by cutting audio in the middle of the first expected chunk with setting low max_new_tokens. <a href="https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-ae43f5490434ac5806d8f435291d94268c759ec8b26cc93cbeacdbb5d31f8c13R419" target="_blank">https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-ae43f5490434ac5806d8f435291d94268c759ec8b26cc93cbeacdbb5d31f8c13R419</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='22'/>

**Comment by sbalandi** [comment]
<p>not required, removed</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/#comment-2399318950'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sbalandi** [comment]
<p>I have checked, it is because of changed audio, with old one numbers are like previous values </p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/#comment-2395869434'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb (hunk 1 above)
Comment (sbalandi) [reply notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb:13]: <p>removed</p>

Comment (sbalandi) [reply notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb:13]: <p>changed</p>

**Comment by sbalandi** [comment]
notification about allowed audio length have added to gradio_helper.py for whisper-subtitles-generation.ipynb

**Comment by as-suvorov** [comment]
<p>Long-form audio support in review: <a href="https://github.com/openvinotoolkit/openvino.genai/pull/941" target="_blank">https://github.com/openvinotoolkit/openvino.genai/pull/941</a></p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/#comment-2395869446'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by as-suvorov — APPROVED** [review]

**Comment by sbalandi** [comment]
removed notification about audio length

**Review by eaidova — APPROVED** [review]

**Comment by sbalandi** [comment]
Have added to README ffmpeg requirement:
`This example requires `ffmpeg` to be installed. All other required dependencies will be installed by the notebook itself.</br>`

**Pull request merged**